charts/signoz/templates/otel-collector/deployment.yaml

```diff
@@ -55,7 +55,15 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.otelCollector.podSecurityContext | nindent 8 }}
+      # todo: add k8s-wait-for initContainer here
+      # this initContainer waits for the schema migrator job to finish
       initContainers:
+        - name: "{{ include "otelCollector.fullname" . }}-migrate-init"
```

Comment:
Perhaps have something like `schemaMigrator.enabled` configuration as well which controls all of the section with migration init containers and K8s Job.